clients/line-bot-client-base/src/main/java/com/linecorp/bot/client/base/http/HttpInterceptor.java

```diff
@@ -18,6 +18,18 @@
 
 import java.io.IOException;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import okhttp3.Interceptor;
+import okhttp3.logging.HttpLoggingInterceptor;
+
 public interface HttpInterceptor {
     HttpResponse intercept(HttpChain httpChain) throws IOException;
+
+    static Interceptor loggingInterceptor() {
+        final Logger slf4jLogger = LoggerFactory.getLogger("com.linecorp.bot.client.wire");
+        return new HttpLoggingInterceptor(slf4jLogger::info)
+                .setLevel(HttpLoggingInterceptor.Level.BODY);
```

Comment:
[Q] Why is code similar to ApiClientBuilder's buildLoggingInterceptor() needed here?

Reply:
nice catch @mokuzon , it is duplicated!
I removed it